packages/paste-core/components/inline-code/stories/index.stories.tsx

```diff
@@ -94,6 +95,16 @@ export const DisplayingAPIKey: StoryFn = () => {
   );
 };
 
+export const WithTruncate: StoryFn = () => {
```

Comment:
suggestion: could you also add a story with a truncated inline code within a block of regular paragraph text?